objectranger/objectranger.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package objectranger
+
+import (
+	"context"
+	"io"
+
+	"github.com/spacemonkeygo/monkit/v3"
+
+	"storj.io/common/ranger"
+	"storj.io/uplink"
+)
+
+var (
+	mon = monkit.Package()
+)
+
+// ObjectRanger holds all the data needed to make object downloadable.
+type ObjectRanger struct {
+	p      *uplink.Project
+	o      *uplink.Object
+	bucket string
+}
+
+// NewObjectRanger creates a new object ranger.
+func NewObjectRanger(p *uplink.Project, o *uplink.Object, bucket string) ranger.Ranger {
```

Comment:
Since the package is `objectranger` then `New` would be sufficient, since the fully qualified name is `objectranger.New`.